starter-code/css/main.css

```diff
@@ -7,4 +7,87 @@ light blue: #1c5380
 body {
   font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
   margin: 0;
+  background-image: url("../images/bgtile.png");
+  background-repeat: repeat-x;
 }
+
+body {
+  font-family: Arial;
+  font-size: 15px;
+  margin: 0;
+  padding: 0;
+  text-align: center;
+}
+.header {
+  background: #00D1AE;
+  height: 30px;
+  margin-bottom: 10px;
+  min-width: 725px;
+  padding-top: 10px;
+  width: 100%;
+}
+.general-container {
+  margin: 5% 10% auto;
+
+}
+.left-bar {
+
+  float: left;
+ /* height: 390px;*/
+  padding-top: 10px;
+  /*width: 20%;*/
+}
+.main-content {
+
+  float: left;
+  /*height: 590px;*/
+  margin: 0 0 0 10%;
+  padding-top: 10px;
+
+  width: 48%;
+}
+
+#text-and-badges {
+  clear:both;
+  background-image: url("../images/frame.png");
+  background-repeat: no-repeat;
+  padding: 10%;
+}
```

Comment:
you may have a problem when the content is bigger than the background image size, to avoid that, check the "background-size: cover;" property, it should rescale the image depending on the width of the container